docs/frameworks/symfony.md

```diff
@@ -127,8 +143,28 @@ Since Symfony 4, the production parameters are configured through environment va
 provider:
     environment:
          APP_ENV: prod
+         TRUSTED_PROXIES: '127.0.0.1'
+         TRUSTED_HOSTS: '^api\.example\.com$'
 ```
 
 The secrets (e.g. database passwords) must however not be committed in this file.
 
 To learn more about all this, read the [environment variables documentation](/docs/environment/variables.md).
+
+## Getting the user's IP
+
+If your application needs the user's IP you must fetch it from the `LAMBDA_CONTEXT` variable.
+Modify `public/index.php` accordingly.
```

Comment:
Would it make sense to forward those values automatically in Bref itself?

I.e. maybe (not sure) set a header with the value that is in the lambda context?

Reply:
Many of these solutions were discussed in https://github.com/brefphp/bref/pull/501

However, I do think it makes sense to cherry-pick useful variables. Ie creating `LAMBDA_CONTEXT_IDENTITY_SOURCEIP`

Reply:
Right, but beyond #501 (which is about the request context containing many things), I'm focusing especially on HTTP-related stuff here: if we have the clients' IP address, we could maybe "fix" the HTTP headers (in the FPM runtime only) so that they contain the correct values out of the box?